### DISCUSSION TITLE
[archived] "Daemon Containers Walk Through" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#9893] [General]
# argo-workflows/walk-through/daemon-containers/

https://argoproj.github.io/argo-workflows/walk-through/daemon-containers/

**Comment by SanjayArsenal** [comment]
Is there way to send output parameter from a daemon container ?

**Reply by agilgur5** [reply]
This was asked [on Slack](https://cloud-native.slack.com/archives/C01QW9QSSSK/p1696746143324949?thread_ts=1696746143.324949&cid=C01QW9QSSSK) recently too. Same as the answer from another contributor there, this is not really possible since daemon steps run until the Workflow finishes.
